corehq/apps/app_manager/models.py

```diff
@@ -3303,15 +3304,14 @@ class ReportAppConfig(DocumentSchema):
     header = DictProperty()
     graph_configs = DictProperty(ReportGraphConfig)
     filters = SchemaDictProperty(ReportAppFilter)
-    uuid = StringProperty(default='')
+    uuid = StringProperty(required=True)
 
     _report = None
 
-    @classmethod
-    def wrap(cls, obj):
-        # todo: can remove once existing ReportAppConfig have been migrated
-        _soft_assert_uuid(obj.get('uuid'), 'Check uuids for apps containing UCR %s' % obj.get('report_id'))
-        return super(ReportAppConfig, cls).wrap(obj)
+    def __init__(self, *args, **kwargs):
+        super(ReportAppConfig, self).__init__(*args, **kwargs)
+        if not self.uuid:
+            self.uuid = random_hex()
```

Comment:
and I think the reason I'm asking is that `__init__` functions of db models are pretty big hotspots that we should make sure are well tested